File: net/pdfjet/TextLineWithOffset.cs
using System;

namespace PDFjet.NET {
    internal class TextLineWithOffset {
        internal String textLine;   // A single line of text
        internal float xOffset;     // The horizontal offset (from the X coordinate)

        // Constructor with clear parameter names
        internal TextLineWithOffset(String textLine, float xOffset) {
            this.textLine = textLine;
            this.xOffset = xOffset;
        }
    }
}

File: Sources/PDFjet/TextLineWithOffset.swift


class TextLineWithOffset {
    var textLine: String;   // A single line of text
    var xOffset: Float;     // The horizontal offset (from the X coordinate)

    // Constructor with clear parameter names
    init(_ textLine: String, _ xOffset: Float) {
        self.textLine = textLine;
        self.xOffset = xOffset;
    }
}
